.github/workflows/node-create-package-file/action.yml

```
@@ -50,7 +56,7 @@ runs:
               # set the version
               export package_version="${{ inputs.release_version }}"
               # set the package name
-              export pkg_name="${name}-${node_os}-${node_arch}"
+              export pkg_name="${name}-${node_os}-${node_arch}${MUSL_FLAG}"
```

minor- I think that the libc flag should be next to the node_os, not to the node_arch:
```suggestion
              export pkg_name="${name}-${node_os}${MUSL_FLAG}-${node_arch}"
```

---

If we look at rust way of doing it its "x86_64-unknown-linux-gnu" etc.
My initiative was to follow the same structure rust use. Do you think its better even tho its differ from rust?